connector/connect/server/src/main/scala/org/apache/spark/sql/connect/utils/ErrorUtils.scala

```diff
@@ -40,7 +42,15 @@ import org.apache.spark.sql.connect.service.SparkConnectService
 import org.apache.spark.sql.internal.SQLConf
 
 private[connect] object ErrorUtils extends Logging {
-  private def allClasses(cl: Class[_]): Seq[Class[_]] = {
+  private[connect] def allThrowables(t: Throwable): List[Throwable] = {
+    if (t == null) {
+      List.empty
+    } else {
+      t :: allThrowables(t.getCause)
```

Comment:
What about suppressed throwables?

Reply:
I took a look but didn't find a use case. Do we actually use suppresed throwables in Spark? If not, I think we can suppress them for now?